src/flynt/api.py

```diff
@@ -82,7 +82,8 @@ def default_result():
         for l in unified_diff(contents.split('\n'), new_code.split('\n'), fromfile=filename):
             print(l)
     else:
-        with open(filename, "w", encoding="utf-8") as f:
+        ln = _auto_detect_line_ending(filename)
```

Comment:
Could we use already-read `contents` of the file instead of reading it again?

Reply:
Good point! 😄